tests/Clave.Expressionify.Generator.Tests/CodeGeneratorTests.cs

```diff
@@ -73,7 +84,49 @@ public partial class Nested
             private static System.Linq.Expressions.Expression<System.Func<int, int>> Foo_Expressionify_0 { get; } = (int x) => 8;
         }
     }
-}", TestName = "File scoped namespace")]
+}", TestName = "Nested class")]
```

Comment:
I personally find these test cases a bit hard to read, but that's not specifically something introduced here. Tried a bit with TestCaseSource, but never got to anything I was happy with.